spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/SimpleHostRoutingFilter.java

```diff
@@ -185,9 +177,66 @@ public Object run() {
 		return null;
 	}
 
+	protected PoolingHttpClientConnectionManager newConnectionManager() {
```

Comment:
newConnectionManager()
- Switched from deprecated ThreadSafeClientConnManager to PoolingHttpClientConnectionManager (thread-safe)
- Switched from deprecated SchemeRegistry to RegistryBuilder
- Switched from deprecated SSLSocketFactory to SSLConnectionSocketFactory

newClient()
- Switched from deprecated DefaultHttpClient to CloseableHttpClient 
- Switched from deprecated HttpParams to RequestConfig class.